mobile/src/modules/auth/screens/HomeScreen.tsx

```diff
@@ -122,10 +122,10 @@ class Home extends React.PureComponent<TProps, State> {
         <FullHeightView withoutPaddings>
           <AccountsListHeader priceApi={priceApi} accounts={accounts}/>
           <View>
-            <HomeStackedAreaChart
+            {accounts.length && <HomeStackedAreaChart
               priceApi={priceApi}
               accounts={accounts}
-            />
+            /> || null}
```

Comment:
also hides the chart if no accts, looks much better. let me know if this can be improved...

Reply:
IIRC `|| null` can be omitted...